### PR TITLE
WMI service metric

### PR DESCRIPTION
**Pull request by thiagoftsm**
This PR is bringing metrics to monitor Windows Services;

Screenshot_20221031_022225

### CODE REVIEW

**Review by ilyam8 — CHANGES_REQUESTED** [review]
- The aggregated charts will be broken.
- Aggregation over time will be broken.

If we encode the state/status as an integer.

**Comment by ilyam8** [comment]
@thiagoftsm I don't think we can merge as it is now because of the aforementioned reasons.

File: modules/wmi/charts.go [hunk 1]
```diff
@@ -69,7 +69,14 @@ const (
 	prioProcessesPoolBytes
 	prioProcessesThreads
 
-	prioServiceState
+	prioServiceStateContinuePending
+	prioServiceStatePausePending
```
Comment (ilyam8) [inline modules/wmi/charts.go:73]: I don't think this will do. My suggestion is:
- a chart per service
- the chart dimensions - states.

Comment (thiagoftsm) [reply modules/wmi/charts.go:73]: this will be unusable, because we will have hundred/thousands  of charts.. :/

Comment (ilyam8) [reply modules/wmi/charts.go:73]: what you are doing is unusable too

Comment (thiagoftsm) [reply modules/wmi/charts.go:73]: We need to do something like we are doing right now with Linux services.

Comment (ilyam8) [reply modules/wmi/charts.go:73]: > we will have hundred/thousands of charts

As I said - ND Cloud UI (overview)

Comment (ilyam8) [reply modules/wmi/charts.go:73]: > We need to do something like we are doing right now with Linux services.

Not possible. Prove me wrong.

Comment (thiagoftsm) [reply modules/wmi/charts.go:73]: Windows has hundred of services, times 12 different status we will have thousands, this is the motive I think this is the safest way to go with Microsoft.

Comment (ilyam8) [reply modules/wmi/charts.go:73]: No. The number of charts == the number of services * the number of metrics. It is not a safe/unsafe question - we don't really have a lot of options here.

**Pull request closed without merging**